<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
  <item row="1" column="0">
    <widget class="QLabel" name="label">
    <property name="text">
      <string>Asset</string>
    </property>
    </widget>
  </item>
  <item row="1" column="1">
    <widget class="QLabel" name="lblAsset">
    <property name="text">
      <string></string>
    </property>
    </widget>
  </item>
  <item row="2" column="0">
    <widget class="QLabel" name="label_3">
    <property name="text">
      <string>Quantity</string>
    </property>
    </widget>
  </item>
  <item row="3" column="0">
    <widget class="QLabel" name="label_4">
    <property name="text">
      <string>Unit Price</string>
    </property>
    </widget>
  </item>
  <item row="4" column="0">
    <widget class="QLabel" name="label_5">
    <property name="text">
      <string>Total Price</string>
    </property>
    </widget>
  </item>
  <item row="2" column="1">
    <widget class="QLabel" name="lblQuantity">
    <property name="text">
      <string></string>
    </property>
    </widget>
  </item>
  <item row="3" column="1">
    <widget class="QLabel" name="lblUnitPrice">
    <property name="text">
      <string></string>
    </property>
    </widget>
  </item>
  <item row="4" column="1">
    <widget class="QLabel" name="lblTotalPrice">
    <property name="text">
      <string></string>
    </property>
    </widget>
  </item>
  <item row="7" column="0">
    <widget class="QLabel" name="label_9">
    <property name="text">
      <string>Signed Partial</string>
    </property>
    </widget>
  </item>
  <item row="7" column="1">
    <widget class="QTextEdit" name="txtSigned">
    <property name="readOnly">
      <bool>true</bool>
    </property>
    </widget>
  </item>
  <item row="5" column="0">
    <widget class="QLabel" name="label_2">
    <property name="text">
      <string>UTXO</string>
    </property>
    </widget>
  </item>
  <item row="5" column="1">
    <widget class="QLabel" name="lblUTXO">
    <property name="text">
      <string></string>
    </property>
    </widget>
  </item>
  <item row="0" column="0">
    <widget class="QLabel" name="label_6">
    <property name="text">
      <string>Type</string>
    </property>
    </widget>
  </item>
  <item row="0" column="1">
    <widget class="QLabel" name="lblType">
    <property name="text">
      <string></string>
    </property>
    </widget>
  </item>
  <item row="6" column="0">
    <widget class="QLabel" name="label_6">
    <property name="text">
      <string>Destination</string>
    </property>
    </widget>
  </item>
  <item row="6" column="1">
    <widget class="QLineEdit" name="txtDestination">
    <property name="text">
      <string></string>
    </property>
    <property name="readOnly">
      <bool>true</bool>
    </property>
    </widget>
  </item>
  <item row="8" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
        <rect>
        <x>10</x>
        <y>250</y>
        <width>371</width>
        <height>32</height>
        </rect>
      </property>
      <property name="orientation">
        <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
    </widget>
  </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
